storage/innobase/btr/btr0btr.cc

```diff
@@ -836,7 +836,7 @@ btr_node_ptr_set_child_page_no(
 	ut_ad(!rec_offs_comp(offsets) || rec_get_node_ptr_flag(rec));
 
 	/* The child address is in the last field */
-	field = rec_get_nth_field(rec, offsets,
+	field = rec_get_nth_field_inside(rec, offsets,
 				  rec_offs_n_fields(offsets) - 1, &len);
```

Comment:
After I merged the commit to 10.3, there are 82 references to the renamed function `rec_get_nth_field_inside()` and only 26 to the new function `rec_get_nth_field()`.
To reduce the size of the patch, please keep the old function name as `rec_get_nth_field()` and introduce a different name (say, `rec_get_nth_cfield()`) for the new function. The `cfield` could be a mnemonic of ‘clustered index field’. I would not call it anything like `rec_get_nth_col()`, because the `nth` is the index field position, not the table column position.